### PR TITLE
fix(appName): get rid of references to appId

### PR DESCRIPTION
**Pull request by faust64**
Superseding winstonjs/winston-syslog#74, drop all references to appId

### CODE REVIEW

File: lib/winston-syslog.js [hunk 1]
```diff
@@ -146,7 +145,7 @@ Syslog.prototype.log = function (level, msg, meta, callback) {
   syslogMsg = this.producer.produce({
     severity: level,
     host:     this.localhost,
-    appName:   this.appId || process.title,
+    appName:   this.appName || process.title,
```
Comment (santigimeno) [inline lib/winston-syslog.js]: I wonder if removing this line would also work, as the `appName` in the `producer` has already been set in the constructor.

File: test/format-test.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ vows.describe('syslog messages').addBatch({
             transport = new winston.transports.Syslog({
               port: PORT,
               type: '5424',
-              appId: 'hello'
+              appName: 'hello'
```
Comment (santigimeno) [inline test/format-test.js]: Maybe adding a copy of this test that sets `app_name` too and see if it also works

**Comment by faust64** [comment]
Indeed, the tests pass removing appName as suggested.
And yes, the `app_name` test also seems to pass.

**Comment by santigimeno** [comment]
Landed with the commits squashed in the commit log reworded in https://github.com/winstonjs/winston-syslog/commit/41329137fde739531b8a0d11f0d2cfa32a88004d.  Thanks!

**Pull request closed without merging**